template/concept-template.adoc

```diff
@@ -0,0 +1,63 @@
+== Concept Template
+
+Every topic must have a title with two equal signs `==` to the right.
+The filename of the topic must include all main words of the title separated by dashes `-`. 
+For example, this template has the `concept-template.adoc` filename.
+
+Do not mix concepts and tasks in one topic.
+
+Concepts can include unordered lists.
+
+* Secure access to remote Docker Engine instances.
+* Control access to Docker commands on a user-by-user basis.
+
+Include notices using the following markup, but avoid using more than one notice of each type per concept.
+Change additional notices into normal text to avoid including too many notices in a document.
+
+[NOTE]
+====
+Use notes to convey optional information.
+====
+
+[TIP]
+====
+Use tips to convey useful shortcuts, workarounds, or best practices.
+====
+
+[CAUTION]
+====
+Use caution notices to convey minor risks to property or individuals.
+Always include the following information.
+
+. What is the risk involved. For example: Changing the networking configuration could make the workloads inaccessible.
+. How to avoid the risk. For example: To ensure connectivity isn't interrupted, do not change the subnet in the configuration.
+====
+
+[WARNING]
+====
+Use warnings to convey serious risks to property or individuals.
+Always include the following information.
+
+. What is the risk involved. For example: Disabling your Enforcers can leave your deployment vulnerable.
+. How to avoid the risk. For example: To minimize the risk to your deployment, don't disable all Enforcers at once.
+====
+
+=== Second Level Heading 1
+
+This is the first subsection of this concept.
+
+To include images, use the followinge markup to specify the path, filename, and the width of the image.
+
+[source]
+```
+image::screenshot-example.png[width=500]
+```
+
+Store the images under the `_graphics` folder of the pertinent guide.
+For example in the CNS SaaS guide, store images under `/sass\admin_guide/_graphics/`  
```

Comment:
typo in saas. Is there an issue with the / in the location path?

Reply:
I would remove the backslash from the path since it can cause many issues. It doesn't seem to break the render here though.